packages/api-derive/src/assets/all.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright 2017-2022 @polkadot/api-derive authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { FetchedAssetsEntries, FetchedAssetsIddEntries, FetchedAssetsMetadataEntries } from '@polkadot/api-derive/assets/types';
+import type { DeriveApi } from '@polkadot/api-derive/types';
+import type { StorageKey } from '@polkadot/types';
+import type { EventRecord, Hash } from '@polkadot/types/interfaces';
+import type { FrameSystemEventRecord, PalletAssetsAssetDetails, PalletAssetsAssetMetadata } from '@polkadot/types/lookup';
+import type { u32, Vec } from '@polkadot/types-codec';
+
+import { combineLatest, concat, EMPTY, map, Observable, of, switchMap, take } from 'rxjs';
+
+export interface Asset extends PalletAssetsAssetDetails, PalletAssetsAssetMetadata{
```

Comment:
Looking at other derives, I believe the convention here is to:
* name it `DeriveAsset`
* put it in `./assets/types`.

Also, perhaps some types can be converted to js types instead of scale/substrte. Eg. string for asset name.

Reply:
✅